tests/Makefile

```
@@ -12,11 +12,15 @@ test:
+	$(CC) $(CFLAGS) -o test_systick_init test_systick_init.c led_systick_init_tested.c
+	./test_systick_init
+
 clean:
-	rm -f test_user_switch user_switch_tested.c test_usart2 led_tested.c test_usart2_write led_write_tested.c
-	sed 's/int main(void)/int original_main(void)/g' ../data/led_on_off_print_terminal.c | sed 's/while ((SysTick->CTRL & 0x10000) == 0)/while (mock_systick_check())/g' | sed '1i int mock_systick_check(void);' > led_delay_tested.c
+	rm -f test_user_switch user_switch_tested.c test_usart2 led_tested.c test_usart2_write led_write_tested.c test_systick_init led_systick_init_tested.c
+	sed 's/int main(void)/int original_main(void)/g' ../data/led_on_off_print_terminal.c | sed 's/while ((SysTick->CTRL & SysTick_CTRL_COUNTFLAG_Msk) == 0)/while (mock_systick_check())/g' | sed 's/while ((msTicks - startTicks) < (uint32_t)n)/if (0)/g' | sed '1i int mock_systick_check(void);' > led_delay_tested.c
 	$(CC) $(CFLAGS) -o test_delay test_delay.c led_delay_tested.c
 	./test_delay
 
 clean:
```

checkmake (0.3.2)

[warning] Target "clean" defined multiple times (lines 19 and 25).

(uniquetargets)